Core/Graphics/Include/UWP/CoreWindow/Babylon/GraphicsPlatform.h

```diff
@@ -1,15 +1,17 @@
 #pragma once
 
-#include <Babylon/Graphics.h>
 #include <windows.ui.core.h>
 #include <winrt/windows.ui.core.h>
 #include <winrt/Windows.ApplicationModel.h>
 
-using WindowType = winrt::Windows::UI::Core::ICoreWindow*;
-
-struct Babylon::GraphicsConfiguration
+namespace Babylon
 {
-    WindowType WindowPtr;
-    size_t Width;
-    size_t Height;
-};
+    using WindowType = winrt::Windows::UI::Core::ICoreWindow*;
+
+    struct GraphicsConfiguration
```

Comment:
I don't see too much issue in leaving our config struct in `Babylon` namespace either.

Reply:
Not sure I understand; the struct is in the Babylon namespace here, isn't it? The change is that I'm declaring and implementing it here now, not just implementing it, and that `WindowType` is now in the Babylon namespace, too.

Reply:
This was me agreeing with your change.

Reply:
Gotcha. Yeah, it was less clear to me when looking at the PR here on GitHub, but in the code I could see that the `Babylon` namespace was a better fit than the `Babylon::Graphics` namespace -- partly because there isn't a `Babylon::Graphics` namespace. 😆